FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Ankit on 30/07/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
`internal` is a default access level, no need to declare it explicitly.
Also this class is not intended to be subclassed thus it should be `final`.

Please remove `internal` modifier and make the class `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```